### DISCUSSION TITLE
How to control the cluster radiu of add_points_from_xy?

### DISCUSSION THREAD

**Discussion by zxdawn** [opengeos/leafmap#416] [Q&A] [answered]
Here's the default view of my point data:

<img width="1321" alt="image" src="https://user-images.githubusercontent.com/30388627/231475485-8e3a6f0b-0971-40bd-bb56-b0ce296af2e2.png">

The cluster radius is too large to see some details. For example, if I zoom in the map, there're many dots in different places:
<img width="1301" alt="image" src="https://user-images.githubusercontent.com/30388627/231475668-81ecc93b-70ad-4b6c-bd99-bf23753be709.png">

Is it possible to control the default cluster radius manually?

**Comment by zxdawn** [comment]
The demo on [this repository](https://github.com/simonw/datasette-cluster-map) looks great:

<img width="864" alt="image" src="https://user-images.githubusercontent.com/30388627/231477011-460c82c1-1d8f-40ea-ae28-9aa7628447cd.png">

**Comment by giswqs** [comment, marked as answer]
Good suggestion. I have added the `max_cluster_radius` parameter to the function. See https://github.com/opengeos/leafmap/pull/417. Update the package using `leafmap.update_package()` and restart the kernel to take effect.

**Reply by zxdawn** [reply]
Thanks, it works well!
